app/controllers/sessions_controller.ts

```
@@ -0,0 +1,47 @@
+     * Handle form submission for the create action
+     */
+    async store({ request, response }: HttpContext) {
+      const data = request.all()
+      const payload = await createSessionValidator.validate(data)
+      const email = payload.email?.toLowerCase()
+      const username = payload.username?.toLowerCase()
+      const password = payload.password
+
+      if (email && username) {
+        return response.badRequest({ message: "Either email or username must be provided.", status: "e_missing_identifier" })
+      } else if (email && username) {
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)